jf_agent/__init__.py

```diff
@@ -48,6 +48,15 @@ def write_file(outdir, filename_prefix, compress, results):
         with open(f'{outdir}/{filename_prefix}.json', 'w') as outfile:
             outfile.write(json.dumps(results, indent=2, default=str))
 
+def write_status_file(outdir, type, status):
+    with open(f'{outdir}/status.json', 'r') as status_file:
+        status_json = json.load(status_file)
+        body = {
+            'type': type,
+            'status': status
+        }
+        status_json.append(body)
+        write_file(outdir, 'status', False, status_json)
```

Comment:
What happens when the status file already exists from a prior agent run? We end up with multiple statuses for the same type?

Reply:
Whenever the agent is run, a new directory with the timestamp is created and that is where it dumps the files it writes to. I'm not sure it will end up writing to an already existing status file - but I may be wrong.